### DISCUSSION TITLE
Solis Daily battery discharge always greater than Solis Daily Battery charge

### DISCUSSION THREAD

**Discussion by ocarrolj** [hultenvp/solis-sensor#251] [General]
**Describe the bug**
While using the integration HA consistently shows more energy being provided by the batteries than was used to charge the batteries.  It seems very odd but is also the same in Solid cloud so probably not a bug. I'm curious as to how you're meant to interpret this. My only explanation is that the discharge value actually includes the solar generation? This would point to significant system losses however. 

**Config**
Normal out-of-the-box configuration

**Versions**
HA Version Core 2023.1.2
HACS version 1.29.1
Integration Version c3.2.2 

**Screenshots**
![image](https://user-images.githubusercontent.com/6312011/211538235-61875492-a9cb-4524-957d-8206d75c1f5c.png)
![image](https://user-images.githubusercontent.com/6312011/211538302-310cfec0-9900-491d-9f49-939d11a3d92f.png)
![image](https://user-images.githubusercontent.com/6312011/211538532-691cc55f-dc3f-4abc-adc3-634c3f85afd2.png)

**Comment by hultenvp** [comment]
This integration only passes on the data it gets from the API, save for a few bugs in the API where the sign if off or a digit is off.
These kind of issues I'd really suggest you to take up with Solis and see what they have to tell about how the parameters are specified or whether this points to an issue.

I cannot help you fixing this. Changing it into a discussion topic.

Cheers
